### DISCUSSION TITLE
Work with nii files instead of nii.gz - Optimisation

### DISCUSSION THREAD

**Discussion by dalvSilvermann** [Project-MONAI/MONAILabel#1207] [Ideas]
Dear colleagues,

1. Is it possible to work with Monai Label to train NN with already unpacked nii files?
Because the unpack inside Monai takes to much time, while GPU is in smth like "Stand by mode" and waiting of CPU :0)

2. Maybe we can do it by changing some configs?

![2022-12-13 10 52 47](https://user-images.githubusercontent.com/108551361/207257967-62115f44-fe89-4f61-9283-4d53a8c23e33.jpg)

**Comment by diazandr3s** [comment]
Hi @dalvSilvermann,

Thanks for this.
MONAI Label does support .nii files as well. Here are the configs showing the file format it supports for different apps (i.e. radiology, pathology and endoscopy):  https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/config.py#L58-L67

Hope this helps,
